docs/universalprofilecloud/wallet.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: '🪙 Wallet & Assets'
+sidebar_position: 2
+---
+
+# Wallet & Assets
+
+import React from 'react'
+import LoomVideo from '@site/src/components/LoomVideo'
```

Comment:
```suggestion
```

Reply:
Yes there are Loom videos embedded in this page